Vagrantfile

```diff
@@ -0,0 +1,38 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/precise64"
+  config.vm.provider "hyperv"
+  config.vm.provision :shell, privileged: false, inline: <<-SHELL
+	sudo apt-get update
+
+	# Install pyenv prerequisites
+	sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+	libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+	xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+	# Install pyenv
+	git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+	echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+	echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+	echo 'eval "$(pyenv init -)"' >> ~/.profile
+	export PYENV_ROOT="$HOME/.pyenv"
+	export PATH="$PYENV_ROOT/bin:$PATH"
+	eval "$(pyenv init -)"
+	pyenv install 3.6.6
+	pyenv global 3.6.6
```

Comment:
Why use old version of python? I assume this is what you happened to be using on your machine so you knew it'd work

Reply:
Upgraded 3.8.3